apps/home/blocks.py

```diff
@@ -103,3 +103,94 @@ class TextWithImageBlock(blocks.StructBlock):
     class Meta:
         template = 'apps_home/blocks/text_with_image_block.html'
         icon = 'arrow-down'
+
+
+# teaser blocks
+
+class TeaserBlockCentered(blocks.StructBlock):
+
+    link = blocks.PageChooserBlock(
+        target_model=['apps_home.DetailPage',
+                      'apps_gruenbuch.GruenbuchIndexPage'],
+        help_text='Make sure that teaser title and teaser intro '
+                  'of the chosen page are set.'
+    )
+    link_text = blocks.CharBlock(max_length=50, label='Link Text')
+    background_color = ColorChoiceBlock(
+        help_text='Not choosing a colour will result in a CTA '
```

Comment:
The helptext is a bit wrong, as this is no CTA block. Maybe we can come up with a helptext that works for all blocks and change it everywhere? Maybe just "Not choosing a colour will result in a block with a white background."?